src/components/templates/Poster/Poster.tsx

```diff
@@ -0,0 +1,169 @@
+  };
+
+  const {
+    activeParticipants,
+    passiveListerens,
+
+    isMeActiveParticipant,
+
+    turnVideoOff,
+    turnVideoOn,
+  } = usePosterVideo(venue.id);
+
+  const isPosterLive = venue.isLive;
+
+  const setVenueLiveOn = () => {
+    setVenueLiveStatus(venue.id, true);
+  };
+
+  const setVenueLiveOff = () => {
+    setVenueLiveStatus(venue.id, false);
+  };
+
+  const videoParticipants = activeParticipants.map((participant) => (
+    <VideoParticipant
+      participant={participant}
+      key={participant.identity}
+      additionalClassNames="poster__video-participant"
+    />
+  ));
+
+  const hasFreeSpace = videoParticipants.length < POSTER_CELL_COUNT_MAX;
+
+  return (
+    <>
+      <div className="poster">
+        <div className="poster__header">
+          <div />
+
+          <div className="poster__header--middle-cell">
+            <p className="poster__title">{venue.title}</p>
+            <div className="poster__categories" />
+          </div>
+
+          <div className="poster__header--right-cell">
+            {isMeActiveParticipant && (
+              <div
+                className="poster__control poster__control--stop"
+                onClick={turnVideoOff}
+              >
+                <FontAwesomeIcon icon={faStop} size="lg" />
+                <span className="poster__control-text">Stop video</span>
+              </div>
+            )}
+
+            <OverlayTrigger
+              trigger="click"
+              placement="bottom-end"
+              overlay={
+                <Popover
+                  id="popover-basic"
+                  className="poster__settings-popover"
+                >
+                  <Popover.Content className="poster__settings-popover-content">
+                    <Toggler
+                      type="checkbox"
+                      className="switch-hidden-input"
+                      checked={!!isPosterLive}
+                      onChange={isPosterLive ? setVenueLiveOff : setVenueLiveOn}
+                      title={
+                        isPosterLive ? "Poster is live" : "Make poster live"
+                      }
+                    />
+                  </Popover.Content>
+                </Popover>
+              }
+              rootClose={true}
+            >
+              <div className="poster__control">
+                <FontAwesomeIcon icon={faCog} size="lg" />
+                <span className="poster__control-text">Settings</span>
+              </div>
+            </OverlayTrigger>
+
+            <div className="poster__control">
+              <FontAwesomeIcon icon={faShare} size="lg" />
+              <span className="poster__control-text">Share</span>
+            </div>
+
+            {venue.introVideoUrl && (
+              <div className="poster__control" onClick={showIntroVideoModal}>
+                <FontAwesomeIcon icon={faTv} size="lg" />
+                <span className="poster__control-text">Intro Video</span>
```

Comment:
The other controls only seem to capitalize the first letter of the first word of their label. We should keep this consistent, one way or the other.

```suggestion
                <span className="poster__control-text">Intro video</span>
```